clearwater_config_access/rph_json_config_plugin.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+"""This contains the rph json subclass of ConfigType"""
+
+import subprocess
+import logging
+from metaswitch.clearwater.config_manager.config_type_class_plugin import ConfigType
+LOG_DIR = "/var/log/clearwater-config-manager"
+log = logging.getLogger("cw-config.validate")
+
+class RphJson(ConfigType):
+    def __init__(self, configpath):
+        """ Initialise the class variables. """
+        self.configfile = configpath
+
+        self.name = 'rph_json'
+        self.filetype = 'json'
+
+        # file_download_name is used to agree with the current naming system
+        # when writing to file
+        self.file_download_name = 'rph.json'
+
+        # This help_info appears as user-visible help text in the usage statement
+        # for cw-config.
+        self.help_info = ('''rph_json - maps different resource priority header
+                values to an internal priority value.''')
```

Comment:
Can you make the alignment here a bit nicer, please.

Reply:
Leaving the alignment as it was because of the way the help text is formatted.
(It does the second line off the amount of space you put in, and this amount of space prints it in a sensible place.)